packages/temporal/go.mod

```diff
@@ -3,15 +3,14 @@ module github.com/AdonisEnProvence/MusicRoom
 go 1.16
 
 require (
-	github.com/Devessier/brainy v0.0.10
+	github.com/Devessier/brainy v0.0.11
 	github.com/bojanz/httpx v0.0.0-20201111190843-d1cf01c49b2e
 	github.com/bxcodec/faker/v3 v3.6.0
 	github.com/go-playground/universal-translator v0.17.0 // indirect
 	github.com/go-playground/validator v9.31.0+incompatible
 	github.com/golang/mock v1.6.0 // indirect
 	github.com/gorilla/handlers v1.5.1
 	github.com/gorilla/mux v1.8.0
-	github.com/joho/godotenv v1.3.0 // indirect
```

Comment:
We should manage this issue for next new repos. With for example a js lib dotenv